src/app/common/stores/applications/SsvWeb/Wallet.store.ts

```diff
@@ -141,13 +143,15 @@ class WalletStore extends BaseStore implements Wallet {
     if (walletCondition) {
       await this.onboardSdk.walletSelect(selectedWallet);
       await this.onboardSdk.walletCheck();
+      await this.getFeeRecipientAddress();
```

Comment:
Don't mix the logic of this method with other not related logic.
1. Put this call into `walletHandler` method
2. Rename `walletHandler` to `onWalletConnectedCallback`

Reply:
done